.github/workflows/code-analysis.yml

```diff
@@ -10,10 +10,10 @@ jobs:
   php-stan:
     runs-on: ubuntu-latest
     steps:
-     - name: Install PHP 8.1
+     - name: Install PHP 8.0
```

Comment:
Has the supported version of PHP changed?

Reply:
Yes, It's because some of the dependencies does not support php 8.1, which is why we add 8.0
as a supported version. All the dependencies works with 8.0.

Reply:
What about Php 7.4 ?